native/dpotf2.go

```diff
@@ -0,0 +1,66 @@
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dpotrf computes the cholesky decomposition of the symmetric positive definite
+// matrix a. If ul == blas.Upper, then a is stored as an upper-triangular matrix,
+// and a = U^T U is stored in place into a. If ul == blas.Lower, then a = L L^T
+// is computed and stored in-place into a. This is the unblocked version of the
+// algorithm. Dpotrf returns 0 if the decomposition is successful, otherwise
```

Comment:
I don’t feel particularly strongly, except that I do think the mat64 version has a nice API. I can put it back if you prefer.

> On May 7, 2015, at 7:07 PM, Dan Kortschak <notifications@github.com> wrote:
> 
> In native/dpotf2.go <https://github.com/gonum/lapack/pull/11#discussion_r29912787>:
> 
> > @@ -0,0 +1,66 @@
> > +package native
> > +
> > +import (
> > +	"math"
> > +
> > +	"github.com/gonum/blas"
> > +	"github.com/gonum/blas/blas64"
> > +)
> > +
> > +// Dpotrf computes the cholesky decomposition of the symmetric positive definite
> > +// matrix a. If ul == blas.Upper, then a is stored as an upper-triangular matrix,
> > +// and a = U^T U is stored in place into a. If ul == blas.Lower, then a = L L^T
> > +// is computed and stored in-place into a. This is the unblocked version of the
> > +// algorithm. Dpotrf returns 0 if the decomposition is successful, otherwise
> OK. We'll leave as bool.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gonum/lapack/pull/11/files#r29912787>.
>